application-forum-ui/src/main/resources/ForumCode/Actions.xml

```diff
@@ -65,26 +67,34 @@
         $response.setStatus(403)
       #end
     #end
-  #elseif("$!request.action" == "addtopic")
+  #elseif("$!request.action" == 'addtopic')
     #if("$!request.title.trim()" != '')
       #if("$!request.space" != '')
-        #set($newTopicName = $request.page)
         #set($topicTitle = "$!request.title.trim()")
-        #set($topicDoc = $xwiki.getDocument($newTopicName))
+        #set($topicSpace = "$!request.space")
```

Comment:
As far as I read it, this should actually be `$forumSpace` since `$request.space` is the space of the forum.